tensorboard/plugin_util.py

```diff
@@ -150,6 +151,15 @@ def markdowns_to_safe_html(markdown_strings, combine):
     return warning + sanitized_combined
 
 
+def context(environ):
```

Comment:
Do we need this helper if we already have `context.from_environ()` itself, and that is public (assuming that the plugin is expected to have access to the `RequestContext` type itself, which is in the same module)?

Reply:
I like having this helper so that plugins don’t need to import yet
another TensorBoard utility module. Plugins already import this module
to get `plugin_util.experiment_id(environ)`, and having `context`
adjacent to that feels pretty natural to me.

Reply:
Fair enough, as discussed this gives us a bit more flexibility later if we want to move `RequestContext`.  (I do wish ultimately we had a slightly less "util" name than `plugin_util` but we don't have to fix that now.)